content/en/docs/topics/observing-spinkube.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Observing SpinKube
+description: How to view telemetry data from your Spin apps running in SpinKube.
+weight: 13
+---
+
+This topic guide shows you how to configure SpinKube so your Spin apps export telemetry data to OpenTelemetry collector and Jaeger.
+
+## Prerequisites
+
+Please ensure you have the following tools installed before continuing:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) - the Kubernetes CLI
+- [Helm](https://helm.sh) - the package manager for Kubernetes
+- [SpinKube](https://www.spinkube.dev/docs/spin-plugin-kube/installation) - the Kubernetes plugin for Spin
```

Comment:
I would copy the prerequisites format from [here](https://www.spinkube.dev/docs/topics/using-a-key-value-store/)